api/wallet/spec/definitions/DigitalWallet.yaml

```
@@ -1,25 +1,23 @@
-description: Данные цифрового кошелька
+---
+description: Digital wallet data
 type: object
 required:
   - id
   - provider
 properties:
   id:
-    description: Идентификатор цифрового кошелька
+    description: Digital wallet ID
     type: string
     minLength: 1
     maxLength: 100
     example: zu3TcwGI71Bpaaw2XkLWZXlhMdn4zpVzMQ
   provider:
-    description: Провайдер электронных денежных средств
+    description: Digital wallet service provider
     x-rebillyMerge:
-      - $ref: '#/definitions/DigitalWalletProvider'
+      - $ref: "#/definitions/DigitalWalletProvider"
   token:
-    description: Строка, содержащая данные для авторизации операций над этим кошельком
+    description: A string containing authorization data for transactions on this wallet
```

Раз везде уточнено до этого про цифровой кошелек
```suggestion
    description: A string containing authorization data for transactions on this digital wallet
```